hs_access_control/models/user.py

```diff
@@ -1597,6 +1597,19 @@ def __check_share_resource_with_user(self, this_resource, this_user, this_privil
         """
         return self.__check_share_resource(this_resource, this_privilege, user=this_user)
 
+    def can_copy_resource(self, resource):
+        """
+        Checks whether a user has permissions to copy a resource
+        :param resource: resource to check permissions
+        :param user: user to check resource permissions for
+        :return: Boolean: True if the resource grants the necessary permissions to the user for copying
+        """
+        if resource.raccess.public:
+            return True
+        if self.user.uaccess.can_view_resource(resource):
+            return True
+        return False
```

Comment:
I believe ```can_view_resource``` already check resource.raccess.public, so I don't think we need this new ```can_copy_resource``` method. Just checking ```can_view_resource``` should be sufficient.

Reply:
you're right, I'll make that update to get rid of this method and just use can_view_resource